### PR TITLE
Create BootCDN.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

**Comment by J0WI** [comment]
I tried to collect all the subdomains, but they are all refused.
I don't really understand this service, all these domains seems to redirect or proxy to the original site?

koa.bootcss.com
less.bootcss.com
codeguide.bootcss.com
job.bootcss.com
jquery.bootcss.com
open.bootcss.com
v4.bootcss.com
www.bootcss.com
sass.bootcss.com
jekyll.bootcss.com
gruntjs.bootcss.com
v2.bootcss.com
docs.bootcss.com
w3schools.bootcss.com
v3.bootcss.com
wenda.bootcss.com
v4.bootcss.com
expo.bootcss.com


On the only working cdn.bootcss.com I run into mixed content issues.

File: src/chrome/content/rules/BootCDN.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Mismatch:
+	  *bootcdn.cn
```
Comment (J0WI) [inline src/chrome/content/rules/BootCDN.xml]: www.bootcdn.cn
blog.bootcdn.cn

**Comment by renyouguo** [comment]
@J0WI Where is MCB?

**Comment by J0WI** [comment]
https://cdn.bootcss.com/bootstrap-tokenfield/0.11.8/package/index.html

**Comment by renyouguo** [comment]
@J0WI Anything else?

**Comment by J0WI** [comment]
That's the only MCB page I can find.

**Comment by J0WI** [comment]
Thanks for adding this!

**Pull request merged**